examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+extern crate alloc;
+extern crate lang_items;
+extern crate libtock_drivers;
+
+use core::fmt::Write;
+use libtock_core::result::CommandError;
+use libtock_drivers::console::Console;
+use libtock_drivers::nfc::NfcTag;
```

Comment:
I'm not entirely sure what to do here. Can we gate the whole file by `cfg(feature = "with_nfc")` (and provide a dummy empty example when the feature is off)?